test/Geometry/geometry_icarus_test.cxx

```diff
@@ -97,25 +100,29 @@ int main(int argc, char const** argv) {
   // testing environment setup
   //
   IcarusGeometryTestEnvironment TestEnvironment(config);
+  using namespace lar::standalone;
+  auto const wireReadoutAlg =
+    SetupReadout<geo::WireReadoutSorterICARUS, icarus::ICARUSWireReadoutGeom>(wireReadoutConfig(TestEnvironment),
+                                                                              TestEnvironment.Geometry());
+  auto const auxDetGeom = SetupAuxDetGeometry(TestEnvironment.ServiceParameters("AuxDetGeometry"));
   
   //
   // run the test algorithm
   //
   
-  // 1. we initialize it from the configuration in the environment,
-  geo::GeometryTestAlg Tester(TestEnvironment.TesterParameters());
+  // 1. we initialize it from the environment,
+  geo::GeometryTestAlg Tester(TestEnvironment.Geometry(),
+                              wireReadoutAlg.get(),
+                              auxDetGeom.get(),
+                              TestEnvironment.TesterParameters());
   
-  // 2. we set it up with the geometry from the environment
-  Tester.Setup(*TestEnvironment.Geometry());
```

Comment:
Do you now "Setup" the Tester anymore?

Reply:
The `TestEnvironment.Geometry()` argument is now passed in to the constructor instead of having a separate `Tester.Setup(...)` call.